pkg/transport/generic_transport_controller.go

```diff
@@ -203,6 +209,7 @@ func (c *genericTransportController) Run(ctx context.Context, workersCount int)
 	c.logger.Info("starting workers", "count", workersCount)
 	// Launch workers to process Binding
 	for i := 1; i <= workersCount; i++ {
+		i := i // gotta love go loop variables
```

Comment:
can you name the variable workerId instead of `i := i`? 
that is very confusing for code readers.

Reply:
Code readers that do not know about go loop variables and do not want to learn.

Reply:
I changed the variable name and put a real explanation in the comment.